daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -562,6 +582,12 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
   verify()
 }
 
+object ValidatorPatterns {
+  val NoArgsPattern: Regex = "(.+?)".r.anchored
+  val MultiArgsPattern: Regex = "(.+?)=(.+?=.+[,]?)+".r.anchored
```

Comment:
Supporting typesafe config allows us to avoid rolling our own multi-arg support here which would get complicated and opinionated in favor of a more standard approach (env and .conf files).